packages/TorneloScoresheet/src/pages/GraphicalRecording/GraphicalRecording.tsx

```diff
@@ -21,14 +91,18 @@ const GraphicalRecording: React.FC = () => {
   return (
     <>
       {graphicalRecordingMode && (
-        <View>
-          <ChessBoard
-            positions={graphicalRecordingMode.board}
-            onMove={onMove}
-            flipBoard={
-              graphicalRecordingMode.currentPlayer === PlayerColour.Black
-            }
-          />
+        <View style={styles.mainContainer}>
+          <View style={{ height: 100 }}>
```

Comment:
Inline styling here as well - let's put this in `style.ts`

Reply:
this is just a placeholder though, will be replaces by the player cards once we implement it

Reply:
True! My bad

Reply:
Should probably have a TODO/FIXME there but doesn't matter - I'll approve now